transformations/transformations/pool_allocator.py

```diff
@@ -730,7 +764,7 @@ def inject_pool_allocator_into_calls(self, routine, targets):
         for call in FindInlineCalls().visit(routine.body):
             if call.name.lower() in [t.lower() for t in targets]:
                 parameters = call.parameters
-                call_map[call] = call.clone(parameters=parameters + new_vars)
+                call_map[call] = call.clone(parameters=parameters + new_arguments)
```

Comment:
Can we have the `args` vs. `kwargs` problem for `InlineCalls` as well?

Reply:
Agreed. Should be changed now.